### PR TITLE
Landing carousel

### PR DESCRIPTION
**Pull request by FathimaSayeeda**
Transitions not done. Fine tunings may be required.

### CODE REVIEW

File: vestido/vestido-storefront/components/landing-view/Carousel.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+            <div className={`font-extralight text-${slide.textColor}`}>{slide.subtitle2}</div>
+            <div><Button className="mt-6 h-12 rounded-none text-white bg-[#48CAB2] uppercase font-bold px-10">{slide.buttonText}</Button></div>
+          </div>
+         
+          <Image
+            className="embla__slide__img"
+            src={slide.backgroundImage}
+            alt="Your alt text"
+          />
+        </div></CarouselItem>
```
Comment (fahimalizain) [inline vestido/vestido-storefront/components/landing-view/Carousel.tsx:78]: Please move this to dedicated SubComponent

File: vestido/vestido-storefront/components/landing-view/Carousel.tsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+import {
+  Carousel,
+  CarouselContent,
+  CarouselItem,
+} from "libs/shadcn-ui/src/ui/carousel"
+import Image from "next/image"
+import { Button } from "@vestido-ecommerce/shadcn-ui/button"
+
+const slides = [
+  {
```
Comment (fahimalizain) [inline vestido/vestido-storefront/components/landing-view/Carousel.tsx:15]: Please type this `Array<LandingCarouselItem>`

File: vestido/vestido-storefront/pages/styles.css [hunk 3]
```diff
@@ -71,11 +71,11 @@
...
 @layer base {
-  * {
+  /* * {
     @apply border-border;
-  }
+  } */
   body {
-    @apply bg-background text-foreground;
+    /* @apply bg-background text-foreground; */
     font-feature-settings: "rlig" 1, "calt" 1;
```
Comment (fahimalizain) [inline vestido/vestido-storefront/pages/styles.css:79]: Please do not commit this now, we have to solve this in another PR.

File: vestido/vestido-storefront/components/landing-view/Carousel.tsx [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+    subtitle1: "don't miss today's featured deals",
+    subtitle2: 'Here to bring your lifestyles to next level',
+    buttonText: 'discover now!',
+    buttonLink: '3',
+    textAlign: 'left',
+    textColor: 'white',
+    textPosition:'right-1/4'
+  }
+]
+export function LandingCarousel() {
```
Comment (fahimalizain) [inline vestido/vestido-storefront/components/landing-view/Carousel.tsx:49]: Please move this to `/modules`

**Review by fahimalizain — CHANGES_REQUESTED** [review]

File: vestido/vestido-storefront/components/landing-view/Carousel.tsx (hunk 4 above)
Comment (FathimaSayeeda) [reply vestido/vestido-storefront/components/landing-view/Carousel.tsx]: Done

File: vestido/vestido-storefront/components/landing-view/Carousel.tsx (hunk 2 above)
Comment (FathimaSayeeda) [reply vestido/vestido-storefront/components/landing-view/Carousel.tsx]: Done

File: vestido/vestido-storefront/pages/styles.css (hunk 3 above)
Comment (FathimaSayeeda) [reply vestido/vestido-storefront/pages/styles.css]: Done

File: vestido/vestido-storefront/components/landing-view/Carousel.tsx (hunk 1 above)
Comment (FathimaSayeeda) [reply vestido/vestido-storefront/components/landing-view/Carousel.tsx]: Done

**Review by mohammedfabinsha — APPROVED** [review]

**Review by fahimalizain — APPROVED** [review]

**Pull request merged**